src/main/resources/templates/snowflake/snowflake-schema-evolution/copy-into.sql.ssp

```diff
@@ -0,0 +1,31 @@
+<%--
+Copyright 2018 phData Inc.
+
```

Comment:
There is a shared template for copy-into. Why aren't we using that here?

Reply:
My mistake. We can use the shared template

Reply:
I found one difference in the COPY INTO  query.
In shared template this is the query used : 
COPY INTO ${table.destinationName}_stg (${unescape(table.columnList(null))})
FROM ( SELECT ${unescape(sourceColumnStr)}
FROM @${configuration.name}_stage/${table.sourceName})
FILE_FORMAT = ( TYPE = PARQUET);

And I copied this file from snowflake-snowpipe-append folder. There this query is used :  
COPY INTO ${table.destinationName} (${unescape(table.columnList(null))})
FROM ( SELECT ${unescape(sourceColumnStr)}
FROM @${configuration.name}_stage/${table.sourceName})
FILE_FORMAT = ( TYPE = PARQUET);

There is difference in the table name. "_stg" is appended in the shared template. So far in all the tests I am able to load data with query present in this file.  I think if I use the shared template the query will fail.